digital_land/specification.py

```diff
@@ -147,6 +147,12 @@ def field_type(self, fieldname):
 
         raise ValueError("unknown datatype '%s' for '%s' field" % (datatype, fieldname))
 
+    def field_parent(self, fieldname):
+        field = self.field[fieldname]
+        if field["parent-field"]:
+            return field["parent-field"]
+        return ""
```

Comment:
I think I'd tend to return `None` in this case rather than empty string. Certainly not a rule but just feels better to mean "no result".

Reply:
I'd normally go with `None` too but noticed `""` had been used in the `field_typology` method [line 153](https://github.com/digital-land/digital-land-python/blob/main/digital_land/specification.py#L153)

Reply:
Sorted this with 43cfbd597

Reply:
And 8cbf333